lib/rubocop/cop/style/raise_args.rb

```diff
@@ -98,9 +98,10 @@ def check_compact(node)
             return if exception.send_type? && exception.first_argument&.hash_type?
 
             add_offense(node, message: format(COMPACT_MSG, method: node.method_name)) do |corrector|
-              replacement = correction_exploded_to_compact(node)
+              if (replacement = correction_exploded_to_compact(node))
+                corrector.replace(node, replacement)
+              end
 
-              corrector.replace(node, replacement)
```

Comment:
It seems that the tests pass without the changes in L101-L103. If this change is necessary, can you add a test for it?

Reply:
That is indeed not needed, I was just defensive about passing nil to the corrector but that seems to work just fine. Thank you.